packit/config/config.py

```diff
@@ -100,20 +103,11 @@ def get_user_config(cls) -> "Config":
         return Config.get_from_dict(raw_dict=loaded_config)
 
     @classmethod
-    def get_from_dict(cls, raw_dict: dict, validate=True) -> "Config":
-        if validate:
-            cls.validate(raw_dict)
-
-        config = Config()
-
-        config.debug = raw_dict.get("debug", False)
-        config.dry_run = raw_dict.get("dry_run", False)
-        config.fas_user = raw_dict.get("fas_user", None)
-        config.keytab_path = raw_dict.get("keytab_path", None)
+    def get_from_dict(cls, raw_dict: dict) -> "Config":
+        from packit.schema import UserConfigSchema
```

Comment:
If there's not a specific reason behind importing from `packit.schema` here (and in other methods), then I'd like to see them [at the top of the files](https://stackoverflow.com/questions/128478/should-import-statements-always-be-at-the-top-of-a-module).

Reply:
i think it is not possible, because of circular imports - config is used in schema and schema is used in config. Was trying avoid it, but i think in current implementation it is best way how to do it

Reply:
ok, sure, then comment it in the code, please